desktop/core/src/desktop/js/apps/storageBrowser/FileChooserModal/FileChooserModal.tsx

```diff
@@ -162,26 +286,45 @@ const FileChooserModal = ({
             handleSearch(event.target.value);
           }}
         />
-        <LoadingErrorWrapper loading={loading}>
-          <Table
-            className="hue-filechooser-modal__table"
-            data-testid="hue-filechooser-modal__table"
-            dataSource={tableData}
-            pagination={false}
-            columns={getColumns(tableData[0] ?? {})}
-            rowKey={r => `${r.path}__${r.type}__${r.name}`}
-            scroll={{ y: '250px' }}
-            rowClassName={record =>
-              record.type === BrowserViewType.file
-                ? classNames('hue-filechooser-modal__table-row', 'disabled-row')
-                : 'hue-filechooser-modal__table-row'
-            }
-            onRow={onRowClicked}
-            locale={locale}
-            showHeader={false}
-          />
-        </LoadingErrorWrapper>
+        {isUploadEnabled ? (
+          <DragAndDrop onDrop={onFilesDrop}>{TableContent}</DragAndDrop>
+        ) : (
+          TableContent
+        )}
       </div>
+      <input
+        ref={uploadRef}
+        type="file"
+        className="hue-importer__source-selector-option-upload"
+        onChange={handleFileUpload}
+        accept={SUPPORTED_UPLOAD_TYPES}
+      />
+      {showCreateFolderModal && (
+        <BottomSlidePanel
+          isOpen={showCreateFolderModal}
+          title={t('Create Folder')}
+          cancelText="Cancel"
+          primaryText={t('Create')}
+          onClose={() => {
+            setShowCreateFolderModal(false);
+            setCreateFolderValue('');
+          }}
+          onPrimaryClick={handleCreate}
+        >
+          {/* TODO: Refactor CreateAndUpload to reuse */}
+          <LoadingErrorWrapper errors={createFolderErrorConfig}>
+            <Input
+              defaultValue={createFolderValue}
```

Comment:
why not just use value={createFolderValue}, we don't need to trigger onChange than?